app/views/pensions/incomeFromOverseasPensions/IncomeFromOverseasPensionsCYAViewHelper.scala

```diff
@@ -45,23 +45,26 @@ object IncomeFromOverseasPensionsCYAViewHelper extends CYABaseHelper {
   }
 
   private def summaryRowForPensionSchemeCodes(incomeFromOverseasPensionsViewModel: IncomeFromOverseasPensionsViewModel, taxYear: Int)
-                                       (implicit messages: Messages) : Seq[Option[SummaryListRow]] = {
-    
-    if (incomeFromOverseasPensionsViewModel.paymentsFromOverseasPensionsQuestion.contains(true)) {
-      incomeFromOverseasPensionsViewModel.overseasIncomePensionSchemes.zipWithIndex.map{
-        case (_, index) =>
-          val countryName = Some(Countries.getCountryFromCodeWithDefault(
-            incomeFromOverseasPensionsViewModel.overseasIncomePensionSchemes(index).countryCode)
-            .toUpperCase
-          )
-          Some(summaryListRowWithStrings(
-            "incomeFromOverseasPensions.cya.overseasPensionSchemes",
-            countryName,
-            //TODO - To change the redirect to the Oveseas Pension page
-            ForeignTaxCreditReliefController.show(taxYear, Some(index)))(messages)
-          )
+                                             (implicit messages: Messages) : Option[SummaryListRow] = {
+
+    if (
+      incomeFromOverseasPensionsViewModel.paymentsFromOverseasPensionsQuestion.contains(true)
+        && incomeFromOverseasPensionsViewModel.overseasIncomePensionSchemes.length > 0
+    ) {
+      val countryNames = for {
```

Comment:
val countryNames = for {
        pensionScheme <- incomeFromOverseasPensionsViewModel.overseasIncomePensionSchemes
        countryCode = pensionScheme.countryCode.getOrElse("N/A")
      } yield {
         Countries.getCountryFromCodeWithDefault(Some(countryCode))
            //TODO: Add 3 digit country code parsing as well.
     }

Replace with
val countryNames = for {
        pensionScheme <- incomeFromOverseasPensionsViewModel.overseasIncomePensionSchemes
      } yield {
         Countries.getCountryFromCodeWithDefault(pensionScheme.countryCode)
       }